Content.Shared/_NF/GameRule/PointOfInterestPrototype.cs

```diff
@@ -46,11 +46,17 @@ public sealed partial class PointOfInterestPrototype : IPrototype
     public int MaximumDistance { get; private set; } = 10000;
 
     /// <summary>
-    /// Components to be added to any spawned grids.
+    ///     Components to be added to any spawned grids.
     /// </summary>
     [DataField]
     public ComponentRegistry AddComponents { get; set; } = new();
 
+    /// <summary>
+    ///     What gamepresets ID this POI is allowed to spawn on.
+    /// </summary>
+    [DataField]
+    public string[] SpawnGamePreset { get; private set; } = [];
```

Comment:
Could this use `ProtoId<GamePresetPrototype>` instead of strings?  You'll get some validation that you aren't using misspelled strings that'll never get chosen.